roles/build_push_container/tasks/main.yml

```diff
@@ -0,0 +1,160 @@
+#   Very slow when pushing multiple tags
+- name: Render varibles and optionally print
+  vars:
+    _rendered_tags: >-
+      {% set out = [] -%}
+      {% if cifmw_build_push_container_git_sha is defined -%}
+        {% set _ = out.append(cifmw_build_push_container_git_sha) -%}
+      {% endif -%}
+      {% if cifmw_build_push_container_patch_number is defined -%}
+        {% set _ = out.append(cifmw_build_push_container_patch_number ~ '-latest') -%}
+      {% endif -%}
+      {{ out }}
+    _rendered_tags_with_overrides: >
+      {{
+        _rendered_tags +
+        (
+          cifmw_build_push_container_tag_override
+          if cifmw_build_push_container_tag_override
+          is not none else []
+        )
+      }}
+    _multi_arch_build: "{{ cifmw_build_push_container_supported_platform | length > 1 }}"
+    _container_manifest_name: "{{ 'localhost/' ~ cifmw_build_push_container_name ~ '-manifest' }}"
+    _podman_qemu_user_static_cmd: >-
+      sudo podman run
+      --rm
+      --privileged
+      {{ cifmw_build_push_container_qemu_user_static_image }}
+      --reset
+      -c yes
+      -p yes
+    _podman_build_cmd: >-
+      podman build
+      --security-opt="label=disable"
+      --platform={{ cifmw_build_push_container_supported_platform | join(',') }}
+      {% if _multi_arch_build -%}
+      --manifest={{ _container_manifest_name }}
+      {% endif -%}
+      --tag={{ cifmw_build_push_container_local_build_tag }}
+      --label="build_push_role=true"
+      --file={{ cifmw_build_push_container_containerfile_path }}
+      {{ cifmw_build_push_container_build_context_path }}
+    _podman_push_container_cmd: >-
+      podman push
+      {% if cifmw_build_push_container_registry_username is not none -%}
+      {% if cifmw_build_push_container_registry_password is not none -%}
+      --creds {{ cifmw_build_push_container_registry_username }}:{{ cifmw_build_push_container_registry_password }}
+      {% endif -%}
+      {% endif -%}
+      {% if not cifmw_build_push_container_registry_tls_verify -%}
+      --tls-verify=false
+      {% endif -%}
+      {{ cifmw_build_push_container_local_build_tag }}
+      {{ cifmw_build_push_container_registry_name }}
+    _podman_push_manifest_cmd: >-
+      podman manifest push
+      {% if cifmw_build_push_container_registry_username is not none -%}
+      {% if cifmw_build_push_container_registry_password is not none -%}
+      --creds {{ cifmw_build_push_container_registry_username }}:{{ cifmw_build_push_container_registry_password }}
+      {% endif -%}
+      {% endif -%}
+      {% if not cifmw_build_push_container_registry_tls_verify -%}
+      --tls-verify=false
+      {% endif -%}
+      {{ _container_manifest_name }}
+      {{ cifmw_build_push_container_registry_name }}
+  block:
+    - name: Create manifest if multi-arch requested
+      when: _multi_arch_build
+      block:
+        - name: Install qemu-user-static for multi arch builds
+          ansible.builtin.command:
```

Comment:
Maybe consider leveraging `ci_script`  instead of the `command` ? (true for all of the occurrences)

That would allow to get proper log files per action.

Reply:
Done

Reply:
Done